code/modules/mob/living/carbon/human/life.dm

```diff
@@ -62,6 +62,11 @@ var/global/list/brutefireloss_overlays = list("1" = image("icon" = 'icons/mob/sc
 	//TODO: seperate this out
 	// update the current life tick, can be used to e.g. only do something every 4 ticks
 	life_tick++
+
+	// This is not an ideal place for this but it will do for now.
+	if(wearing_rig && wearing_rig.offline)
+		wearing_rig = null

```

Comment:
Might be a good thing to try then, because Life() code is pretty cluttered as is, and having this check every Life() for every humanoid does add a tiny bit of overhead.